load-tests/h1_h1_passthrough/src/h1_h1_passthrough.bal

```diff
@@ -17,36 +17,30 @@
 import ballerina/http;
 import ballerina/log;
 
-listener http:Listener securedEP = new(9090, {
-    secureSocket: {
+listener http:Listener securedEP = new (9090,
+    httpVersion = "1.1",
+    secureSocket = {
         key: {
             path: "./security/ballerinaKeystore.p12",
-            password: "ballerina"
+        password: "ballerina"
         }
     }
-});
+);
 
-final http:Client nettyEP = check new("https://netty:8688", {
-    secureSocket: {
+final http:Client nettyEP = check new("https://netty:8688",
+    httpVersion = "1.1",
+    secureSocket = {
         cert: {
             path: "./security/ballerinaTruststore.p12",
             password: "ballerina"
         },
         verifyHostName: false
     }
-});
+);
 
 service /passthrough on securedEP {
-    resource function post .(http:Caller caller, http:Request clientRequest) {
-        http:Response|http:ClientError response = nettyEP->forward("/service/EchoService", clientRequest);
-        if (response is http:Response) {
-            error? result = caller->respond(response);
-        } else {
-            log:printError("Error at h1_h1_passthrough", 'error = response);
-            http:Response res = new;
-            res.statusCode = 500;
-            res.setPayload(response.message());
-            error? result = caller->respond(res);
-        }
+    resource function post .(http:Caller caller, http:Request clientRequest) returns error? {
+        http:Response response = check nettyEP->forward("/service/EchoService", clientRequest);
+        return caller->respond(response);
```

Comment:
Let's return the response rather using caller

Reply:
Fixed with [03ab7a3739f257aa4ac62770af56a7b3eff4804e](https://github.com/ballerina-platform/module-ballerina-http/pull/1166/commits/03ab7a3739f257aa4ac62770af56a7b3eff4804e)